### DISCUSSION TITLE
Copilot does not connect (IntelliJ)

### DISCUSSION THREAD

**Discussion by LordMaddhi** [community/community#36869] [Copilot Conversations]
Hi, 

I have installed Copilot on my IntelliJ and if im signing in i get the message "You're all set up!" but in IntelliJ it doesnt connect and says i have to login.

**Comment by iamcjwilliams** [comment]
@LordMaddhi Can you provide a screenshot of the error you're receiving?

**Comment by LordMaddhi** [comment]
hi, after logging in i get this "congratulations, youre all set!" message but in intelliJ it says i have to log in :/

![chrome_QLYUi0duDK](https://user-images.githubusercontent.com/38781381/197708752-f69375c0-b66e-4eee-90c0-027917d2e00d.png)
![idea64_WSVqBEeoqo](https://user-images.githubusercontent.com/38781381/197708787-85192280-38c7-4c17-b8ee-e96520eddda8.png)
![idea64_EKQYotXIBf](https://user-images.githubusercontent.com/38781381/197708801-095c5619-4af5-4b94-b964-79186dc06e0c.png)

**Comment by Ghilt** [comment]
I have the same behavior. The device connection steps indicate nothing wrong, i can perform the steps over and over again. But Github copilot status always tells me to log in.

**Comment by iamcjwilliams** [comment]
@LordMaddhi @Ghilt A few steps to try: 

1. Try restarting your editor
2. Validate your copilot subscription or trial 
3. If you're still experiencing issues, visit [here](https://intellij-support.jetbrains.com/hc/en-us/articles/207241085-Locating-IDE-log-files) to provide your logs and post a screenshot here.

**Reply by Ghilt** [reply]
Thanks nr 2 was my problem. I thought i had reached the end of the sign up process, but i had not finalised it properly after adding payment method.

**Reply by LordMaddhi** [reply]
Thanks for your reply. 
The logs: `INFO - #copilot - [INFO] [auth] [2022-10-27T14:06:47.291Z] Invalid copilot token: missing token: 403 `

**Reply by LordMaddhi** [reply]
oh, i fixed it with nr 2. i thought my free student copilot is automatic activated. had to do it manually.

**Comment by Moses-Code-Dev** [comment]
https://github.com/djoxy-code/Github-copilot-Fix
